rpm/ngfd-tone-generator.conf

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+  <!-- Allow any connection to receive the message -->
+  <policy context="default">
```

Comment:
This still applies -- can we make it so that only root or someone can own these names?